src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -743,33 +746,37 @@ def import_model(
         framework_name="transformers", framework_versions=framework_versions
     )
 
+    if trust_remote_code:
+        logger.warning(
+            "trust_remote_code is set to True. Bentoml will load the specified model into memory by default. To avoid loading the model to memory, try setting the keyword argument clone_repository=True."
+        )
+
     config = transformers.AutoConfig.from_pretrained(
         model_name_or_path,
         proxies=proxies,
         revision=revision,
         force_download=force_download,
         resume_download=resume_download,
+        trust_remote_code=trust_remote_code,
         **extra_hf_hub_kwargs,
     )
 
     is_auto_class = False
-    if pretrained_model_class is None:
-        if getattr(config, "architectures", None):
-            try:
-                pretrained_model_class = getattr(transformers, config.architectures[0])
-                logger.info(
-                    f"pretrained_model_class is not provided, bentoml will create a model with the following pretrained model class {config.architectures[0]}. Available pretrained classes for this model: {config.architectures}."
-                )
-            except AttributeError:
-                logger.info(
-                    "Config architecture does not exist in transformers module."
-                )
-        if getattr(config, "auto_map", None):
-            for auto_class in config.auto_map:
-                if auto_class.startswith("AutoModel"):
-                    pretrained_model_class = getattr(transformers, auto_class)
-                    is_auto_class = True
-                    break
+    pretrained_model_class = None
+    if getattr(config, "architectures", None):
+        try:
+            pretrained_model_class = getattr(transformers, config.architectures[0])
+            logger.info(
+                f"pretrained_model_class is not provided, bentoml will create a model with the following pretrained model class {config.architectures[0]}. Available pretrained classes for this model: {config.architectures}."
+            )
+        except AttributeError:
+            logger.info("Config architecture does not exist in transformers module.")
```

Comment:
no need to info log about this. just pass. we can raise an example if both config.auto_map and architectures is not found